discord/src/commonMain/kotlin/work/socialhub/planetlink/discord/action/DiscordMapper.kt

```
@@ -0,0 +1,212 @@
+    }
+
+    // ---------------------------------------------------------------- //
+    // Utility
+    // ---------------------------------------------------------------- //
+
+    /** Discord timestamps are ISO8601 strings. */
+    fun parseTimestamp(timestamp: String?): Instant? {
+        if (timestamp.isNullOrEmpty()) return null
+        return try {
+            Instant.parse(timestamp)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)